pubnub-kotlin/pubnub-kotlin-impl/src/main/kotlin/com/pubnub/internal/endpoints/presence/HereNowEndpoint.kt

```diff
@@ -27,8 +31,23 @@ class HereNowEndpoint internal constructor(
     override val channelGroups: List<String> = emptyList(),
     override val includeState: Boolean = false,
     override val includeUUIDs: Boolean = true,
+    override val limit: Int = MAX_CHANNEL_OCCUPANTS_LIMIT,
+    override val offset: Int? = null,
 ) : EndpointCore<Envelope<JsonElement>, PNHereNowResult>(pubnub), HereNow {
     private val log: PNLogger = LoggerManager.instance.getLogger(pubnub.logConfig, this::class.java)
+    private val effectiveLimit: Int = if (limit in 1..MAX_CHANNEL_OCCUPANTS_LIMIT) {
```

Comment:
We say that it could be between `0` and `infinity` (so far as specified in the current codebase), but here we limit it with `1` as the lower limit. We shouldn't let users specify smaller than `0` and larger than the upper limit.

Reply:
But we should accept `0` as `limit`. The current condition won't let us do it, right?

Reply:
ok, makes sense. I will allow 0 as limit.